packages/webdriverio/src/utils/interception/devtools.js

```diff
@@ -25,15 +25,17 @@ export default class DevtoolsInterception extends Interception {
                     continue
                 }
 
+                request.statusCode = responseStatusCode
```

Comment:
@christian-bromann we can use this one to have `statusCode` check in the `expect`.

Reply:
sounds good to me